telemetry/otel-collector-config.yaml

```diff
@@ -1,9 +1,26 @@
+# https://opentelemetry.io/docs/collector/configuration/
+
+# How data gets into the Collector
 receivers:
+  # Data sources: traces, metrics, logs
   otlp:
     protocols:
+      http:
+        endpoint: "0.0.0.0:4318"
       grpc:
+        endpoint: "0.0.0.0:4317"
+  # Data sources: metrics
+  prometheus:
+    config:
+      scrape_configs:
+        - job_name: otel-collector
```

Comment:
Please use the same `job_name` structure as we are in FGA. This was named intentionally based on the recommendation of OTEL.

The job name should be a fully qualified name composed of a namespace and service name descriptor (e..g `fga/openfga`). And would you mind adding the `service` label here?

https://opentelemetry.io/docs/reference/specification/compatibility/prometheus_and_openmetrics/#resource-attributes-1 for more info.